v3/moondrop/dash75/dash75_r1_v1.5.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Moondrop Dash75",
+    "vendorId": "0x6DD4",
+    "productId": "0x7501",
+    "firmwareVersion": 1,
```

Comment:
```suggestion
```
Theres no firmware version define in the qmk side so please remove this as well to avoid confusion.

Reply:
I'm confused?  [This](https://github.com/qmk/qmk_firmware/pull/20890/files#diff-4ff2a5dc5a0572b2dc4f198a699f3f053033e2d8965d67d0887320c45b43ef86R6) not it?
![image](https://github.com/the-via/keyboards/assets/366673/d4efa754-2729-4d1a-b3bc-99426d6ca2f4)

Reply:
Ah i didn't look in there sorry. Was looking at the KB level.

Reply:
For the key color I'm following the guidelines of the [KLE JSON Rules](https://www.caniusevia.com/docs/layouts#kle-json-rules) section on the docs.  It states:
![image](https://github.com/the-via/keyboards/assets/366673/7e949972-a197-4b03-8b32-479fdb44bc5b)

Modifiers are in turn are:
![image](https://github.com/the-via/keyboards/assets/366673/05442fd9-b311-4024-8866-8f3bb4b9302c)
*(From [Wikipedia](https://en.wikipedia.org/wiki/Modifier_key) )*

Thus neither [Tab] nor [Backspace] are modifiers as they function to add (or remove) content form type, and not "modify" a secondary key.  *(Tab is most cases operates as 8x Spaces)*

Is this also a misinterpretation on my behalf?

Reply:
Not a misinterpretation from Wikipedia but I guess in the kb world for a long time we always have specified modifiers to include tab and backspace, keys that are not part of the alphas. if you look at keysets and how they're coloured this is supposed to reflect that with the themes that you can apply in VIA.

Reply:
Ok ... I'll change them.